docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Great Expectations overview
+---
+
+import CheckpointFlowchart from './images_overview/checkpoint_flowchart.png'
+
+This guide is intended for new users of Great Expectations (GX) and those looking for a more foundational understanding of GX, its components, and its principal workflows.  You will be introduced to the moving parts of GX and how they interact without digging through the code that governs those processes and interactions.  The insight provided in this guide will help you understand the how and why of processes you work through in future tutorials.
+
+## Great Expectations (GX)
+
+GX is a Python library that provides a framework for describing the acceptable state of data and then validating that the data meets those criteria. 
+
+GX enables you to build trust in your data's quality by testing how it performs against your expectations.  Reusable components and ongoing testing provide visibility into the health of your data pipeline over time. The human-readable reports generated by GX allow you to share the results of that analysis with others and better collaborate on improvements to both your data and your understanding of your data.
+
+With GX providing insight into your data and helping assure its quality, you can ensure that your data is fully suitable to be used as intended in your processes.
+
+## GX core components
+
+When working with GX from end to end - that is, from installation of GX to the validation of your data - there are four core components that you will use.  These are "top level" components that interface, contain, or manage underlying objects and processes to make your work easier.  This guide will go over each of these four components and what they govern in more depth after this section. However, in brief, the four components are:
+
+- **Data Context:** Manages the settings and metadata for a GX project, as well as providing an entry point to the GX Python API.
+- **Datasources:** tell GX how to connect to your source data, and how to organize the retrieved data for future use.
+- **Expectations:** tell GX the standards to which your data should conform.
```

Comment:
```suggestion
- **Expectations:** Identifies the standards to which your data should conform.
```

Reply:
suggestion implemented; it'll be in the next commit.